### PR TITLE
The high voltage from tasers can now set people with fire stacks (so things like dousing in fuel) on fire

### PR DESCRIPTION
**Pull request by silicons**
Obviously don't fullmerge this

or do i don't care

### CODE REVIEW

**Comment by deathride58** [comment]
unironically a good change tho

File: code/modules/projectiles/projectile/energy/stun.dm [hunk 1]
```diff
@@ -23,6 +23,7 @@
 		var/mob/living/carbon/C = target
 		SEND_SIGNAL(C, COMSIG_ADD_MOOD_EVENT, "tased", /datum/mood_event/tased)
 		SEND_SIGNAL(C, COMSIG_LIVING_MINOR_SHOCK)
+		C.ignite_mob()
```
Comment (Ghommie) [inline code/modules/projectiles/projectile/energy/stun.dm]:
```suggestion
		C.IgniteMob()
```
You had one job.

**Review by nik707 — APPROVED** [review]
actually fullmerge this tho

**Comment by ghost** [comment]
really do tho

**Pull request merged**